packages/fxa-admin-server/src/test/lib/sentry.spec.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import 'reflect-metadata';
+
+import { assert } from 'chai';
+import { GraphQLError } from 'graphql';
+import 'mocha';
+import { Logger } from 'mozlog';
+import proxyquire from 'proxyquire';
+import sinon, { SinonSpy } from 'sinon';
+import { StubbedInstance, stubInterface, stubObject } from 'ts-sinon';
+
+const sandbox = sinon.createSandbox();
+
+describe('sentry', () => {
+  let logger: Logger;
+  let mockCaptureException: any;
+  let reportGraphQLError: any;
+
+  beforeEach(async () => {
+    const mockScope = { setContext: sinon.stub() };
+    const mockCapture = (func: any) => {
+      func(mockScope);
+    };
+    mockCaptureException = sinon.stub();
+    reportGraphQLError = proxyquire('../../lib/sentry.ts', {
+      '@sentry/node': { withScope: mockCapture, captureException: mockCaptureException }
+    }).reportGraphQLError;
+    logger = stubInterface<Logger>();
+  });
+
+  afterEach(() => {
+    sandbox.reset();
+  });
+
+  it('captures an error', async () => {
+    const originalError = new Error('boom');
+    const err = new GraphQLError(
+      'Internal server error',
+      undefined,
+      undefined,
+      undefined,
+      ['resolver', 'field'],
+      originalError
+    );
+    reportGraphQLError(false, logger, err);
+    const errResult = mockCaptureException.args[0][0];
+    assert.equal(errResult.message, 'boom');
+    const logSpy = (logger.error as unknown) as SinonSpy;
+    assert.isTrue(logSpy.calledOnceWith('graphql', { path: 'resolver.field', error: 'boom' }));
```

Comment:
Could additionally assert that mockScope.setContext have been called with the expected arguments.

Reply:
Added that.